lib/imapSimple.js

```diff
@@ -92,6 +92,40 @@ ImapSimple.prototype.openBox = function (boxName, callback) {
     });
 };
 
+/**
+ * Close a mailbox
+ *
+ * @param {boolean} [autoExpunge=true] If autoExpunge is true, any messages marked as Deleted in the currently open mailbox will be remove
+ * @param {function} [callback] Optional callback, receiving signature (err)
+ * @returns {undefined|Promise} Returns a promise when no callback is specified, resolving to `boxName`
+ * @memberof ImapSimple
+ */
+ImapSimple.prototype.closeBox = function (autoExpunge=true, callback) {
```

Comment:
what is the minimum node version required to use default params like this?

Reply:
derp nevermind, i am the one that suggested this and commented that it is node 6. sorry about that.

Reply:
6.4.0
https://node.green/#ES2015-syntax-default-function-parameters-basic-functionality